Sources/Tagged/Tagged.swift

```diff
@@ -56,7 +58,21 @@ extension Tagged: Equatable where RawValue: Equatable {
   }
 }
 
-extension Tagged: Error where RawValue: Error {
+extension Tagged: Error where RawValue: Error {}
+
+extension Tagged: LocalizedError where RawValue: LocalizedError {
```

Comment:
this overrides all of the properties of LocalizedError.

We could also override `localizedError` here, and it work, but I dont think its desirable (if `errorDescription` is nil, `localizedDescription` returns default value which includes the type name. Keeping the name of the tagged type present in this case is imho better than overriding to include only the name of the untagged type). Thoughts?